gui/src/main/index.ts

```diff
@@ -1391,6 +1391,16 @@ class ApplicationMain {
         );
       }
     });
+    IpcMainEventChannel.windowsSplitTunneling.handleDeleteApplication((application) => {
+      if (windowsSplitTunneling) {
+        this.guiSettings.deleteBrowsedForSplitTunnelingApplications(application.absolutepath);
+        return windowsSplitTunneling.removeApplicationFromCache(application);
+      } else {
+        throw Error(
+          'windowsSplitTunneling.handleRemoveApplication function called without being imported',
+        );
+      }
```

Comment:
The `else` does not seem necessary.

```suggestion
      throw Error(
        'windowsSplitTunneling.handleRemoveApplication function called without being imported',
      );
```